### DISCUSSION TITLE
Creating a new Windows & MacOS project with MAUi

### DISCUSSION THREAD

**Discussion by sid-dahiya** [dotnet/maui#184] [General] [answered]
I'm working on a project using .Net Core 3.x currently that we intend to make available on Windows and MacOS. We'd love to use MAUi especially since this is a new project and we have flexibility of choosing the frameworks.

Unfortunately, I've not been able to find any sample projects, and the ones I do manage to stumble upon, focus on iOS and Android. Can someone share information about how we get started with MAUi?

**Comment by Happypig375** [comment, marked as answer]
The first release of MAUI is scheduled November 2021. Meanwhile, you can use http://github.com/xamarin/Xamarin.Forms today.

**Comment by davidortinau** [comment]
Also, here's the guide to setting up a macOS project. https://docs.microsoft.com/en-us/xamarin/xamarin-forms/platform/other/mac
